.github/workflows/sam-smoke-tests.yaml

```
@@ -1,94 +1,27 @@
 name: sam-smoke-tests
 
 on:
-  pull_request:
-    paths-ignore: ['**.md']
-  push:
-    branches:
-      - develop
-    paths-ignore: ['**.md']
+  workflow_call:
+    inputs:
+      sam-hostname:
+        description: 'a generic hostname for Sam'
+        required: true
+        type: string
+      ENV:
+        description: 'test environment (qa, alpha)'
+        required: true
+        type: string
+      auth-service-account-email:
+        description: 'email of auth service account to run smoke test'
```

```suggestion
        description: 'email address of Sam user that will be use for testing authenticated endpoints'
```
My first reaction here is to not call this a `service-account` email, but just an `email`.  The reality for the smoke tests is that they just need an access token for an enabled user on the Sam host that is being tested.  It does not need to be an SA.  Now, in this particular run-context on GHA, you may have particular criteria in the downstream job(s) that require this to be an SA, and that's fine with me.  Whatever you think makes the most sense.

---

Updated the variable name to `smoke-test-user-email` to clearly state its purpose.